src/main/java/org/prebid/server/bidder/adoppler/AdopplerBidder.java

```diff
@@ -60,17 +61,8 @@ public AdopplerBidder(String endpointUrl, JacksonMapper mapper) {
                 final ExtImpAdoppler validExtImp = parseAndValidateImpExt(imp);
                 final String updateRequestId = request.getId() + "-" + validExtImp.getAdunit();
                 final BidRequest updateRequest = request.toBuilder().id(updateRequestId).build();
-                final BidRequest outgoingRequest = updateRequest.toBuilder().imp(Collections
-                        .singletonList(imp)).build();
-                final String body = mapper.encode(outgoingRequest);
-                final MultiMap headers = HttpUtil.headers().add("x-openrtb-version", "2.5");
-                result.add(HttpRequest.<BidRequest>builder()
-                        .method(HttpMethod.POST)
-                        .uri(endpointUrl)
-                        .headers(headers)
-                        .payload(outgoingRequest)
-                        .body(body)
-                        .build());
+                final String url = endpointUrl + "/processHeaderBid/" + validExtImp.getAdunit();
```

Comment:
Notice that they used `url.PathEscape` so you need to use appropriate method in HttpUtils for .getAnunit